### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by SaraVieira**
Fixes all linting poroblems in packages/app

### CODE REVIEW

File: packages/app/src/app/components/Preview/DevTools/elements.ts [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import styled, { css } from 'styled-components';
 
+const getCursor = ({ primary, open }) => {
+  if (primary) return 'default';
```
Comment (MichaelDeBoey) [inline packages/app/src/app/components/Preview/DevTools/elements.ts]: Adding curly braces (`{}`) around the body will make this less error-prone.

```suggestion
  if (primary) {
    return 'default';
  }
```

File: packages/app/src/app/components/Preview/DevTools/elements.ts [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+  if (open) return 'row-resize';
```
Comment (MichaelDeBoey) [inline packages/app/src/app/components/Preview/DevTools/elements.ts]: Same

```suggestion
  if (open) {
    return 'row-resize';
  }
```

File: packages/app/src/app/pages/Dashboard/Sidebar/SandboxesItem/FolderEntry/CreateFolderEntry.tsx [hunk 3]
```diff
@@ -25,7 +25,13 @@ interface Props {
   noFocus?: boolean;
 }
 
-export default ({ basePath, teamId, noFocus, close, depth }: Props) => {
+export const CreateFolderEntry = ({
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/Dashboard/Sidebar/SandboxesItem/FolderEntry/CreateFolderEntry.tsx]: Typing this with `FunctionComponent` will give better IDE help.

```ts
export const CreateFolderEntry: FunctionComponent<Props> = ({
  // ...
}) => {
  // ...
};
```

File: packages/app/src/app/pages/Dashboard/Content/Sandboxes/Filters/index.tsx [hunk 4]
```diff
@@ -12,7 +12,11 @@ interface Props {
   hideFilters?: boolean;
 }
 
-export default ({ possibleTemplates, hideOrder, hideFilters }: Props) => (
+export const Filters = ({
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/Dashboard/Content/Sandboxes/Filters/index.tsx]: Typing this with `FunctionComponent` will give better IDE help.

```ts
export const Filters: FunctionComponent<Props> = ({
  // ...
}) => {
  // ...
};
```

File: packages/app/src/app/pages/Dashboard/Content/Sandboxes/Filters/FilterOptions/index.tsx [hunk 5]
```diff
@@ -13,13 +13,13 @@ interface Props {
   signals?: any;
 }
 
-const FilterOptions = ({
+const FilterOptionsComponent = ({
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/Dashboard/Content/Sandboxes/Filters/FilterOptions/index.tsx]: Typing this with `FunctionComponent` will give better IDE help.

```ts
export const FilterOptionsComponent: FunctionComponent<Props> = ({
  // ...
}) => {
  // ...
};
```

File: packages/app/src/app/components/Preview/DevTools/elements.ts [hunk 6]
```diff
@@ -1,5 +1,16 @@
 import styled, { css } from 'styled-components';
 
+const getCursor = ({ primary, open }) => {
```
Comment (CompuIves) [inline packages/app/src/app/components/Preview/DevTools/elements.ts]: Very nice!

**Pull request merged**